libs/math/tests/unit/fixed_point/fixed_point.cpp

```diff
@@ -236,6 +236,129 @@ TEST(FixedPointTest, Conversion_64_64)
               static_cast<double>(fp32_t::TOLERANCE));
 }
 
+TEST(FixedPointTest, FromString_16_16)
+{
+  // Get raw value
+  fp32_t one("1fp32");
+  fp32_t zero_point_five("0.5fp32");
+  fp32_t one_point_five("1.5fp32");
+  fp32_t two_point_five("2.5fp64");
+  fp32_t m_one_point_five("-1.5fp");
+  fp32_t m_one_point_five_em10("-1.5e-3fp32_t");
```

Comment:
em10, but e-3 (not e-10) is it ok?

Reply:
scientific notation was removed, after discussion about fp problems.